src/views/TechnicalAssistance/index.tsx

```diff
@@ -37,6 +37,7 @@ function TechnicalAssistance() {
         way old emails, bookmarks, and browser history isn't messed up)
         */}
         <Redirect from="*advice*" to="*guidance*" />
+        <Redirect from="*recommendations*" to="*insights*" />
```

Comment:
I'm okay with this, but just wondering if it would be better to use the explicit urls instead of wildcards?